core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/io/writeJson.kt

```diff
@@ -61,91 +63,121 @@ internal object SerializationKeys {
     const val TYPES = "types"
 }
 
-// See docs/serialization_format.md for a description of
-// serialization versions and format.
-internal const val SERIALIZATION_VERSION = "2.1.0"
+private val valueTypes =
+    setOf(Boolean::class, Double::class, Int::class, Float::class, Long::class, Short::class, Byte::class)
```

Comment:
Where's Char?

Reply:
ah, but would Char make sense here? or would that just become a String from json?

Reply:
I think second options

Reply:
By default, everything is read as string, kotlinx-serialization doesn't have the char option for JsonPrimitive